### ISSUE TITLE
Personal Videos?

### ISSUE THREAD

**Issue by Oddwin** [TheLogicMaster/SDH-AnimationChanger#28]
can I add videos that are NOT hosted by the in-app browser? 

I want to add a video of our cats for my wife's bootup.

**Comment by TheLogicMaster** [comment]
> can I add videos that are NOT hosted by the in-app browser? 
> 
> I want to add a video of our cats for my wife's bootup for Xmas.

See the README for instructions of manual animation configuration.

**Comment by Oddwin** [comment]
Copy, This is what I feared... Im struggling to understand it. :) Ill keep farting around. Thank you for taking the time.

**Comment by redredkrovy** [comment]
Did you ever solve this issue?  I just downloaded the plugin last night.  I have 80 vids already from a different program that I would like to import and I can't figure out how.  I've added them to the /homebrew/animations directory but the plugin is not detecting them.

**Comment by TheLogicMaster** [comment]
There's no automatic way to import batches of videos at the moment. Currently, a subdirectory needs to be created for each "group" of animations, named as the animation should show up in the plugin, with the suspend, boot, and throbber animations named based on the default names, or remapped using a JSON config file per-directory, as detailed in the README.

**Comment by redredkrovy** [comment]
Ok, so I've gone in and made a folder called vhs_startup.  The whole path is ~/homebrew/animations/vhs_startup.  Inside that folder I have placed vhs_startup.webm.  Inside that folder I have also created config.json with the following information.
`{
"boot": "vhs_startup.webm",
"suspend": "",
"throbber": ""
}`
Then I've gone into game mode, into Decky, and told it to reload the plugin.  It still does not show up.  What am I doing wrong?

**Comment by TheLogicMaster** [comment]
Hmm, that all sounds correct. When you say reloaded, do you mean from the main Decky menu with the list of plugins with options to uninstall them? I'm not sure if restarting plugins works for the backends yet, so restarting the system may be necessary to do that. There's an option to reload the config from the AnimationChanger plugin page, though, and that should work. Otherwise, it may be worth checking the plugin log at `/tmp/animation_changer.log` to see if there are any errors.

**Comment by redredkrovy** [comment]
Ok so I was reloading the plugin through Decky.  That doesn't work obviously.  Neither does clicking "Return to Game Mode" on the desktop.  I either have to restart the deck or click on "Reload Config" in the Animation Changer menu.  After doing that it's detecting the custom vids.

Now I just have to go through and do 80 of them or go all the way back to when SteamDeckRepo went public and redownload them all.

**Comment by CalebBurnham** [comment]
Is there an update to be made to the README? the config file no longer looks as described and the directory where animations are stored has changed. When I try to create a /homebrew/animations directory, the OS (or DeckyLoaader, not sure which) deletes it. I am a software dev so I can probably puzzle it out but for future users I think an updated manual animation setup guide will be useful

**Comment by TheLogicMaster** [comment]
Yeah, I forgot to update the README after updating the plugin to use the new official mechanism for getting configuration and runtime data directories, and I haven't looked into where those actually are in practice. It would still have to say that the mentioned paths are only the defaults on Steam OS, but could vary.
